### ISSUE TITLE
[BUG] "Upgrade your Collaborate Project to Plastic SCM" Dialog keeps popping up in Cloned Project

### ISSUE THREAD

**Issue by chriswalz** [VeriorPies/ParrelSync#70]
**Describe the bug**
Cloned project will have "Upgrade" dialog every time I click Play button

**To Reproduce**
1. Have plastic SCM setup on main unity project
2. Create clone
3. Press play
4. Dialog pops up

**Expected behavior**
No dialog

**Screenshots/Video**

![Capture](https://user-images.githubusercontent.com/6971318/152911130-455e42f4-f017-4c95-aa52-f90d54da835a.PNG)

**Environment (please complete the following information):**
 - Unity Editor Version: (Windows 2020.3.26f1)
 - ParrelSync Version: (1.5.0)
 - OS Version: Windows
 - 
**Additional context**

**Comment by 314pies** [comment]
Didn't have much experience with Plastic SCM...
Anyone else is having the same issue?

**Comment by GrandNoobis** [comment]
> Didn't have much experience with Plastic SCM... Anyone else is having the same issue?

Yes, very annoying and time consuming to turn off after every play... Especially when running more than one clone.

**Comment by mccorkle** [comment]
The package in manifest.json that is removed is `com.unity.collab-proxy` in my build.  I glanced over the code, but I don't see anywhere that Parrelsync modifies the packages in the target clone, but that is what I imagine would be required to resolve this automagically for future users.  

I also opened an issue over in the Unity forums to make Unity (the owner of PlasticSCM now) aware of the issue:
* https://forum.unity.com/threads/plasticscm-forced-upgrade-bug-with-parrelsync.1271885/
